pkg/master/pod_cache.go

```diff
@@ -192,6 +192,7 @@ func (p *PodCache) computePodStatus(pod *api.Pod) (api.PodStatus, error) {
 	} else {
 		newStatus.Info = result.Status.Info
 		newStatus.PodIP = result.Status.PodIP
+		newStatus.Conditions = result.Status.Conditions
```

Comment:
What is the desired behavior? Should the pods be removed from load balancing? I would expect that the pod's condition should become unknown in this case.